serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-custom-dashboard-page.adoc

```diff
@@ -0,0 +1,42 @@
+= Dashboards in Kogito Serverless Workflow Tools extension
+:compat-mode!:
+// Metadata:
+:description: Dashboards in Kogito Serverless Workflow Tools extension
+:keywords: kogito, workflow, serverless, Quarkus, Dev UI, Dashboards
+:dashboard_guide: https://www.dashbuilder.org/docs/#chap-dashbuilder-yaml-guides
+:dashboard_editor: https://start.kubesmarts.org/
+
+In Kogito Serverless Workflow Tools, the Dashboards page is used to display the available dashboard files. The page displays a list of available dashboards and add filters to the list.
+
+.Dashboard page
+image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-dashboard-list.png[]
+
+The table on the Dashboards page displays the following details:
+
+* *Name*: the name of a custom dashboard.
+* *Last Modified*: the time reference indicating when the custom dashboard was last updated.
+
+== Creating a custom dashboard
+
+=== Create a custom dashboard file
+The {dashboard_guide}[dashboard guide] for creating dashboards and visualizations with YAML. You can run all examples with {dashboard_editor}[Dashbuilder YML Online].
```

Comment:
We should probably unify YAML/YML, so mostlikely YAML.
```suggestion
The {dashboard_guide}[dashboard guide] for creating dashboards and visualizations with YAML. You can run all examples with {dashboard_editor}[Dashbuilder YAML Online].
```

Reply:
Done